ui/StatusQ/tests/src/TestHelpers/testmodel.cpp

```diff
@@ -99,6 +102,32 @@ void TestModel::invert()
     emit layoutChanged();
 }
 
+void TestModel::removeEverySecond()
```

Comment:
Can we have cases where rows are added in between the layout changed events?

Reply:
Hmm, good question... I don't know any example where it happens (I was surprised with that one with romovals). It's also hard to imagine circumstances of layout changed with additions.

But... it would be very problematic to handle such event - proxies would be unaware which items are added, and it would be hard to identify them e.g. to take persistent indexes for them. So I hope it's not a valid behavior :sweat_smile: